cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -0,0 +1,304 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package startcmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+const testHostURL = "localhost:8080"
+const testInboundHostURL = "localhost:8081"
```

Comment:
We need to rewrite this, please see what I did  https://github.com/hyperledger/aries-framework-go/pull/317
Probably it will not conflict because the file name was changed but be careful